### ISSUE TITLE
Incorrect indentation suggested for series of functions

### ISSUE THREAD

**Issue by evoL** [sasstools/sass-lint#749]
**What version of Sass Lint are you using?**
1.8.0

**Please include any relevant parts of your configuration**

``` yaml
options:
  merge-default-rules: false

rules:
  indentation: 2
```

**What did you do? Please include the actual source code causing the issue.**

``` sass
@function hello($value)
  @return $value

@function world($value)
  @return $value
```

**What did you expect to happen?**
I expected the lint to pass.

**What actually happened? Please include any error messages given to you by Sass Lint.**

```
test.sass
  5:3  error  Expected indentation of 4 spaces but found 2  indentation

✖ 1 problem (1 error, 0 warnings)
```

It seems that the indentation is detected correctly only for the first function. The next ones will have it incorrect. You can verify this by adding another line to the second function and swap them.

**If you're using a IDE plugin have you tried the CLI too?**
This is from the CLI when ran with `sass-lint --config sass-lint.yml --verbose test.sass`. The file contents are listed above.

**Comment by DanPurdy** [comment]
This is actually due to a bug in the AST at the moment. You can see the issue here https://github.com/tonyganch/gonzales-pe/issues/193 unfortunately it affects more than indentation at the moment.

**Comment by bgriffith** [comment]
This is fixed in the current latest release `v1.11.0`
